src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,37 @@
+import "./Footer.css"
+import logo from "./logo.svg";
+import { BottomNavigation, BottomNavigationAction } from "@mui/material";
+import { Container, Grid, Box, Button, Typography } from "@mui/material";
```

Comment:
Duplicated import 
Both import from "@mui/material"
I think this should be combined into one

Reply:
fixed